http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpDockerDAOSpec.scala

```diff
@@ -18,6 +18,7 @@ class HttpDockerDAOSpec extends AnyFlatSpec with Matchers with BeforeAndAfterAll
     // TODO this will break if AoU moves off Dockerhub and we delete these images
     // dockerhub with tag
     ContainerImage("broadinstitute/terra-jupyter-aou:1.0.17", DockerHub),
+    ContainerImage("kkimler/r4.0_tca:2020_11_09", DockerHub), // added after a bug fix (see ticket IA-2439)
```

Comment:
This image is the problematic user image. I haven't been able to find another DockerHub image with the same characteristics. I'm thinking it's better to have one than none. If the user makes changes and breaks our unit tests, it should be pretty straightforward to remove it then.